### PR TITLE
DE0 Nano SoC porting

### PR DESCRIPTION
**Pull request by Deryugin**
Add support for Terasic  DE0_Nano_SoC Version-A Board.

### CODE REVIEW

File: src/drivers/ambapp/ambapp_id.h [hunk 1]
```diff
@@ -10,8 +10,6 @@
 #ifndef AMBAPP_H_
 #define AMBAPP_H_
 
-#include <module/embox/driver/ambapp_api.h>
```
Comment (anton-bondarev) [inline src/drivers/ambapp/ambapp_id.h]: Why is this required?
If it rm it can break sparc templates

Comment (Deryugin) [reply src/drivers/ambapp/ambapp_id.h]: fixed
